frontend/public/components/filter-toolbar.tsx

```diff
@@ -421,6 +421,14 @@ type RowFilterBase<R> = {
   defaultSelected?: string[];
 };
 
+export type RowSimpleFilter<R = any> = {
```

Comment:
why is this different than `RowSimpleFilter` from `console-types` ?

Reply:
There were all sorts of type errors when I used the definition from console-types in filter-toolbar.  Hopefully having all areas look at console-types, will resolve the errors I was seeing.